tensorflow/python/data/util/convert_test.py

```diff
@@ -18,16 +18,19 @@
 from __future__ import division
 from __future__ import print_function
 
+from absl.testing import parameterized
+
 from tensorflow.python.data.util import convert
+from tensorflow.python.data.kernel_tests import test_base
+from tensorflow.python.framework import combinations
 from tensorflow.python.framework import constant_op
 from tensorflow.python.framework import dtypes
 from tensorflow.python.framework import tensor_shape
-from tensorflow.python.framework import test_util
 from tensorflow.python.platform import test
 from tensorflow.python.util import compat
 
 
-class ConvertTest(test.TestCase):
+class ConvertTest(test_base.DatasetTestBase, parameterized.TestCase):
```

Comment:
All tests should use the `combinations.generate` decorator to identify which combinations of (TF 1 vs TF 2) x (graph vs. eager) to test. In other words, you should add the decorator even to methods that were not previously decorated with `test_util`.

Reply:
addressed in the commit.